TemplateProject/fastlane/Fastfile

```diff
@@ -0,0 +1,95 @@
+fastlane_version "2.44.1"
```

Comment:
The default `fastlane init` includes this which is why it's here. I suppose this could be useful if you're using a global fastlane, but since we've installed it via the Gemfile and specify to use it with `bundle exec fastlane` it _should_ always be compatible. Though that wouldn't catch if someone tried to use it without the bundle exec. So this could be beneficial there to guide them to the correct one.

It would only _have_ to be updated here if we updated the fastlane version to an incompatible one, so I don't think it'd have to be changed that much